packages/sv/lib/cli/create.ts

```diff
@@ -119,7 +122,10 @@ export const create = new Command('create')
 				...initialSteps,
 				`  ${i++}: ${highlight(pmRunCmd)}`,
 				'',
-				`To close the dev server, hit ${highlight('Ctrl-C')}`
+				`To close the dev server, hit ${highlight('Ctrl-C')}`,
+				'',
+				`To recreate this project without prompts, run:`,
+				`  ${pc.cyan(prompt)}`
```

Comment:
When we started this feature we had multiple ideas around where to put this message. But in the "What's next?" section, it's taking too much space!

<img width="1015" height="640" alt="Image" src="https://github.com/user-attachments/assets/1fd79d76-d1c4-44db-aaab-385c356d1a82" />

That's why it was gray, and at a great position.

It's overloading the screen for 100% of users while only a very small % will reuse the cli in the same context.